compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -257,7 +257,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     fn suggest_missing_writer(
         &self,
         rcvr_ty: Ty<'tcx>,
-        args: (&'tcx hir::Expr<'tcx>, &'tcx [hir::Expr<'tcx>]),
```

Comment:
This argument is getting a bit unwieldy, could you break out a small type for it with named fields?